content/docs/fundamentals/create_architecture.md

```diff
@@ -0,0 +1,10 @@
+---
+weight: 340
+title: "Create Architecture"
```

Comment:
I'd prefer gerund in titles, now it looks too much like a call for action "Create!", "Learn!", etc.
```diff
-title: "Create Architecture"
+title: "Creating Architecture"
```

Reply:
I actually had an opposite idea. If this is a guide, we should tell people what they need to do.